book/src/simple_application/drawing.md

```diff
@@ -0,0 +1,172 @@
+
+# Setting Up An ECS System
+
+In this stage, we are going to draw something!
+
+> You will be able to run your game during this stage, but you will need to
+> `Ctrl+c` the terminal window to stop it, as we don't process keyboard events
+> yet.
+
+Firstly, we need to revisit the `State.onStart` trait method.
+
+```rust
+fn on_start(&mut self, 
+            world: &mut World, 
+            assets: &mut AssetManager, 
+            pipe: &mut Pipeline)
+```
+Let's talk about the parameters here. The first one - `&mut World` is the
+global object containing all entities, components, and systems. We can use it to
+add *entities* to the world. These entities can have an arbitrary set of
+*component* information associated with them, The entity is the thing, and the
+components are the bits of data attached to it (or more accurately, the types
+of information). *Systems* then drive the data in the entities for the
+components they are interested in/responsible for. I find it a bit confusing,
+so let's jump into pong to see if a real world example helps.
+
+When we want to add an entity, we call `world.create_now()` which returns a
+[builder pattern][builder_pattern] object. We can then call `with` on this
```

Comment:
I think it would be best to pick a pattern for links and stick to it. I'm happy for that to be 2-char.